Tweaked A Number of things

DARK-DarkSlayer opened the pull request:
- fixed centipede
- added dew to hopperable list, didn't see since dew was inside patron folder
- MLG sniper dmg increased, to be slightly lower than sniper
- Bigbomb price change from
750 coins 200 sulphur to 1250 coins 250 sulphur
- Iron Platform hp slightly reduced, 15 to 12
- Amr projectile range increased to have same range as Gaussrifle
- Machinegun health increased from 8 to 17.

DARK-DarkSlayer committed:
Tweaked A Number of things

- fixed centipede
- added dew to hopperable list, didn't see since dew was inside patron folder
- MLG sniper dmg increased, to be slightly lower than sniper
- Bigbomb price change from
750 coins 200 sulphur to 1250 coins 250 sulphur
- Iron Platform hp slightly reduced, 15 to 12
- Amr projectile range increased to have same range as Gaussrifle
- Machinegun health increased from 8 to 17.

Vam-Jam commented:
Could you please make seperate PR's for multiple balance changes and fixes. I would accept the centipide and dew fix then.

I don't agree with the machine gun (2.2x increase in hp), even more so that you've been using it to spawn camp.

DARK-DarkSlayer commented:
TFlippy approved

DARK-DarkSlayer committed:
Vodka Stackable

Fixed buying 2 dinghy's and tavern with beer/vodka
Made Vodka/Beer Stackable up to 4

DARK-DarkSlayer committed:
Chicken Changes

- Made it so that chance for higher tier chicken is increased depending on how long the map time is drawn out
- Fixed powerdrill and readded it to tinkertable
- Fixed Chickens not dropping coins
- Fixed Civillian/commanderchicken not having their phone/blueprints on them.

DARK-DarkSlayer committed:
Important Fix for My mistake

Just tested in dedicated server/local host
now fixed it

DARK-DarkSlayer committed:
Merge branch 'master' into DarkChanges
